pyautosplit/game.py

```diff
@@ -109,9 +109,19 @@ def get_address(varname):
         for name, var in self.data["variables"].items():
             if var["type"] == "rsp" or var["type"] == "rbp":
                 continue
+
             addr = get_address(name)
             try:
-                if "var_type" in var and var["var_type"] == "bool":
+                if var["type"] == "sptr" or var["type"] == "vptr":
+                    if var["type"] == "sptr":
+                        addr = self.process.base_address + addr
```

Comment:
Why adding the the start of the text segment to addresses? This should not be hardcoded.